From 4f4faface211a72d338aa1acdd651ba0abf94f64 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 17 Nov 2017 10:50:51 +0800
Subject: [PATCH 3068/5242] MLK-16715-2 binding-doc: usb: ci-hdrc-usb2: add
 imx7ulp and imx8qm compatible

commit  763bf11b1d46a5cd1f4ebac5dbeeddd1da700e86 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add imx7ulp and imx8qm compatible.

Acked-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/usb/ci-hdrc-usb2.txt       |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt b/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
index 3b7ba44..0fe8cf1 100644
--- a/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
+++ b/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
@@ -10,6 +10,8 @@ Required properties:
 	"fsl,imx6sx-usb"
 	"fsl,imx6ul-usb"
 	"fsl,imx7d-usb"
+	"fsl,imx7ulp-usb"
+	"fsl,imx8qm-usb"
 	"lsi,zevio-usb"
 	"qcom,ci-hdrc"
 	"chipidea,usb2"
-- 
1.7.9.5

